### PR TITLE
Integration test verifies replicasets are deleted when namespace is deleted

### PR DESCRIPTION
**Pull request by onyiny-ang**
**What this PR does / why we need it:**
Provides integration tests to test:
- Replicasets created in a namespace are deleted when a namespace is deleted
- Events created in a namespace are deleted when a namespace is deleted

**Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):**
Not aware of any

Special notes for your reviewer:
I believe this is complete for testing replicasets but I am still working on events. I'm unsure how to create a test event to be created in a namespace for this purpose.

### CODE REVIEW

File: build-log.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+I1120 12:58:57.047] Bootstrap J...
```
Comment (marun) [inline build-log.txt]: oops - please remove.

File: test/integration/crud_test.go [hunk 2]
```diff
@@ -20,8 +20,15 @@ import (
 	"fmt"
 	"testing"
 
+	apiv1 "k8s.io/api/core/v1"
```
Comment (marun) [inline test/integration/crud_test.go]: kube imports should be grouped with those starting on line 31 (and that group sorted).

File: test/integration/crud_test.go [hunk 3]
```diff
@@ -95,6 +102,67 @@ func TestFederationCRUD(t *testing.T) {
...
+
+		//Create replicaset
+		config := fedFixture.APIFixture.NewConfig()
+		client := fedFixture.APIFixture.NewClient(fmt.Sprintf("crud-test-%s", federatedtypes.ReplicaSetKind))
+		rsAdapter := adapterFactory(client, config, nil)
+		obj := rsAdapter.NewTestObject(nsName)
+
+		//Create replicaset inside of namespace
+		replicasetObj, err := rsAdapter.FedCreate(obj)
+		//		t.Logf("Creating replicaset in namespace %s", namespace.Name)
```
Comment (marun) [inline test/integration/crud_test.go]: Avoid committing commented out code.

**Comment by marun** [comment]
May be worth mentioning your intent of allowing api-only tests to run in both integration and e2e, and that the new tests are rewrites of existing e2e tests.

File: test/integration/crud_test.go [hunk 4]
```diff
@@ -20,22 +20,21 @@ import (
...
-	apiv1 "k8s.io/api/core/v1"
-
-	"k8s.io/api/core/v1"
-
-	"k8s.io/apiserver/pkg/storage/names"
-
 	"github.com/pborman/uuid"
 
+	"k8s.io/api/core/v1"
+	apiv1 "k8s.io/api/core/v1"
```
Comment (marun) [inline test/integration/crud_test.go]: Duplicate import?

File: test/integration/crud_test.go [hunk 5]
```diff
@@ -20,22 +20,21 @@ import (
...
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	pkgruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/storage/names"
 	federationapi "k8s.io/federation/apis/federation/v1beta1"
 	"k8s.io/federation/pkg/federatedtypes"
 	"k8s.io/federation/pkg/federatedtypes/crudtester"
 	"k8s.io/federation/test/integration/framework"
 )
 
+// This test is intended to allow api-only tests to run in both integration and e2e
```
Comment (marun) [inline test/integration/crud_test.go]: I'm assuming this is intended for the follow-on PR?

Comment (onyiny-ang) [reply test/integration/crud_test.go]: yes, I realized this was not at all accurate when I was working on the follow-on PR. I can change it to reflect the current state of things.

**Comment by irfanurrehman** [comment]
Apologies for the delay was OOO.
@onyiny-ang can you please squash and/or reword the commit messages
especially the one with `just trying some things`.

**Pull request closed without merging**